ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -370,6 +369,7 @@ private void planTasks() {
           continue;
         }
         plannedTables.add(table.id());
+        tableRuntime.beginPlanning();
```

Comment:
I suggest using a try finally block to begin planning and clean planning status

Reply:
In the current try catch block, if an exception occurs, the `planning` state will be converted to `pending`. If it is normal, it will be converted to the corresponding state of `optmizing`, such as `major`. If there is no `plan` out of `processtask`, `cleanPendingInput` will be called

Reply:
Make sense